central/rbac/k8srole/search/searcher_impl.go

```diff
@@ -9,12 +9,14 @@ import (
 	"github.com/stackrox/rox/central/role/resources"
 	v1 "github.com/stackrox/rox/generated/api/v1"
 	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
 	"github.com/stackrox/rox/pkg/sac"
 	"github.com/stackrox/rox/pkg/search"
 )
 
 var (
-	k8sRolesSACSearchHelper = sac.ForResource(resources.K8sRole).MustCreateSearchHelper(mappings.OptionsMap)
+	k8sRolesSACSearchHelper   = sac.ForResource(resources.K8sRole).MustCreateSearchHelper(mappings.OptionsMap)
+	k8sRolesSACPgSearchHelper = sac.ForResource(resources.K8sRole).MustCreatePgSearchHelper(mappings.OptionsMap)
```

Comment:
The Postgres search helper does not actually use the `OptionsMap` passed in input here. For the sake of correctness, it would be good to use the options map from the postgres schema.

Reply:
@rhybrillou Nice catch! Tnx! If `optionsMap` is not used in `pgSearchHelper`, should we consider refactoring that, or it's planned to be done later?